numba/tests/test_ir.py

```diff
@@ -431,6 +436,7 @@ def check_diffstr(string, pointing_at=[]):
                 b[idx], b[idx + 1] = b[idx + 1], b[idx]
                 break
 
+        assert change_set
```

Comment:
use `self.assertTrue` instead

Reply:
can do, this isn't actually part of the testing, it's ensuring that the state of the thing to test is actually valid.